stories/backstory.md

```diff
@@ -0,0 +1,22 @@
+# Application backstory
+
+One of the guys at work, Derek, is some kind of VIM protegy. He works in
+tmux, because it is "the only way to code". He switches between
+workspaces so rapidly that he gave the product person with epilepsy a
```

Comment:
😬

Reply:
I know this was in the original version and I couldn't get rid of it. It's just too good!